engineyard/ey-cookbooks-dev-v7
# [FBZ-10444] Migrate Memcached Recipe to v7

**Description of your patch**
Adds support for Memcached by migrating recipe from v6 to v7

**Recommended Release Notes**
Added support for Memcached

**Estimated risk**
Medium

**Components involved**
* `cookbooks/ey-memcached`
* `custom-cookbooks/memcached`
* `cookbooks/ey-app/recipes/create.rb`
* `cookbooks/ey-base/metadata.rb`
* `cookbooks/ey-base/recipes/custom.rb`
* `cookbooks/ey-collectd/metadata.rb`

**Description of testing done**

* Create a new environment with v7 as stack
* Add a utility instance named as memcached / or update `ey-memcached/attributes/default.rb` to indicate which instance  to install/test memcached
* Upload recipe for cookbooks/ey-memcached and custom-cookbooks/memcached
* Add environment variable key `EY_MEMCACHED_ENABLED` with the value of `true`
* SSH into the instance and execute `systemctl status memcached.service`, it should display the status for `memcached.service`

**QA Instructions**

* Create a new environment with v7 as stack
* Add a utility instance named as memcached / or update `ey-memcached/attributes/default.rb` to indicate which instance  to install/test memcached
* Upload recipe for cookbooks/ey-memcached and custom-cookbooks/memcached
* Add environment variable key `EY_MEMCACHED_ENABLED` with the value of `true`
* SSH into the instance and execute `systemctl status memcached.service`, it should display the status for `memcached.service`
